tests/CMakeLists.txt

```
@@ -126,6 +134,9 @@ if (UNIX
     add_subdirectory(report)
     add_subdirectory(SampleApp)
     add_subdirectory(SampleAppCRT)
+    if (${ENABLE_ZERO_BASE_TESTS} MATCHES "Yes")
```

This is a bool option, just evaluate the variable itself.
```suggestion
    if (ENABLE_ZERO_BASE_TESTS)
```